src/modules/previewpane/UnitTests-PreviewHandlerCommon/UnitTests-PreviewHandlerCommon.csproj

```diff
@@ -105,7 +107,7 @@
       <Version>1.3.2</Version>
     </PackageReference>
     <PackageReference Include="System.Threading.Tasks.Extensions">
-      <Version>4.5.3</Version>

```

Comment:
Why not push everything to the higher, newer version?

Reply:
Test using Moq is throwing exception in case of updating to newer version
![image](https://user-images.githubusercontent.com/15176579/76696138-5ac72a80-6645-11ea-87da-b9c748c16d2c.png)

This is because of the assembly version change in `System.Threading.Tasks.Extensions` in newer nuget version. I did tried rerunning the tests with Assembly Redirect in that case the CI pipeline is failed with a different exception.
![image](https://user-images.githubusercontent.com/15176579/76696509-7df3d900-6649-11ea-96da-6f72a42b6073.png)

I'm going to retry to trigger CI pipeline by simply removing both Unit tests project that are using Moq.  And resolve the issue of failing unit tests as a separate issue.